index.html

```diff
@@ -0,0 +1,385 @@
+          </select><p>
+          <label for="f9roll2" id="l9r2">Enter roll 2</label>
+          <select name="f9roll2" id="f9roll2">
+            <option value=""></option>
+            <option value="0">0</option>
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+          </select>
+          <div id="f9score"> </div>
+        </div>
+      </div>
+      <div class = "grid-item">
+        <div class = "dropdown" id="frame10">
+          <label for="f10roll1" id="l10r1">Enter roll 1</label>
+          <select name="f10roll1" id="f10roll1">
+            <option value=""></option>
+            <option value="0">0</option>
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+          </select><p>
+          <label for="f10roll2" id="l10r2">Enter roll 2</label>
+          <select name="f10roll2" id="f10roll2">
+            <option value=""></option>
+            <option value="0">0</option>
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+          </select><p>
+          <label for="f10roll3" id="l10r3">Enter roll 3</label>
+          <select name="f10roll3" id="f10roll3">
+            <option value=""></option>
+            <option value="0">0</option>
+            <option value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+          </select>
+          <div id="f10score"> </div>
+          <button type="button" name="button" id="go">Go!</button>
+        </div>
+      </div>
+    </div>
+    <button type="button" name="button" id="reset">Reset</button>
+
+  </body>
+</html>
```

Comment:
HTML interface works and looks neat and tidy. 

Only idea I had is, there must be a way to remove duplication of the code...not sure how this would be done in HTML. With html.erb you can use partials - not sure if there is a similar way in Javascript.